cmd/ncproxy/ncproxy.go

```diff
@@ -559,6 +559,42 @@ type ttrpcService struct {
 	m sync.Mutex
 }
 
+func getComputeAgentClient(agentAddr string) (computeagent.ComputeAgentService, error) {
+	conn, err := winio.DialPipe(agentAddr, nil)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to connect to compute agent service")
+	}
+	client := ttrpc.NewClient(
+		conn,
+		ttrpc.WithUnaryClientInterceptor(octtrpc.ClientInterceptor()),
+		ttrpc.WithOnClose(func() { conn.Close() }),
+	)
+	return computeagent.NewComputeAgentClient(client), nil
+}
+
+func (s *ttrpcService) reconnectComputeAgents(ctx context.Context) {
+	contents, err := agentStore.getKeyValueMap(ctx)
+	if err != nil {
+		log.G(ctx).WithError(err).Warn("failed to get compute agent information")
+	}
+	s.m.Lock()
+	defer s.m.Unlock()
+
+	for k, v := range contents {
```

Comment:
Is the key container ID and value is the addr? Can we change `k, v` to `containerID, addr` then? (or anything more descriptive).

Reply:
I changed it to `cid` and `addr`